common/source/docs/common-prearm-safety-checks.rst

```diff
@@ -42,13 +42,13 @@ Pre-arm checks that are failing will also be sent as messages to the GCS while d
     3D Accel calibration needed                             Accelerometer calibration has not been done         Complete the :ref:`accel calibration <common-accelerometer-calibration>`
     Accels calibrated requires reboot                       Autopilot must be rebooted after accel calibration  Reboot autopilot
     Accels inconsistent                                     Two accelerometers are inconsistent by 0.75 m/s/s   Re-do the :ref:`accel calibration <common-accelerometer-calibration>`.  Allow autopilot to warm-up and reboot.  If failure continues replace autopilot
-    Accels not healthy                                      At least one accelerometer is not providing data    Reboot autopilot.  If failure continues replace autopilot
     ADSB out of memory                                      Autopilot has run out of memory                     Disable features or replace with a higher powered autopilot
+    Accels not healthy                                      At least one accelerometer is not providing data    Reboot autopilot.  If failure continues replace autopilot
     AHRS: not using configured AHRS type                    EKF3 is not ready yet and vehicle is using DCM      If indoors, go outside.  Ensure good GPS lock.  Check for misconfiguration of EKF (see :ref:`AHRS_EKF_TYPE<AHRS_EKF_TYPE>`)
     AHRS: waiting for home                                  GPS has not gotten a fix                            If indoors, go outside.  Ensure compass and accelerometer calibrations have been completed.  Eliminate radio-frequency sources that could interfere with the GPS
     Airspeed 1 not healthy                                  Autopilot is unable to retrieve data from sensor    Check the physical connection and :ref:`configuration <airspeed>`
     AP_Relay not available                                  Parachute misconfigured                             Parachute is controlled via Relay but the relay feature is missing. `Custom build server <https://custom.ardupilot.org/>`__ was probably used, rebuild with relay enabled
-    Auxiliary authorisation refused                         External system has disallowed aurhorisation        Check external authorisation system
+    Auxiliary authorization refused                         External system has disallowed authorization        Check external authorisation system
```

Comment:
Well, this is just american spelling vs UK spelling.. I vote for leaving it with an "s".

Reply:
okay...found some more errors...but not UK vs US